index.js

```diff
@@ -67,6 +90,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			read.on('data', function() {
+				if (progress.enabled) {
+					var written = write.bytesWritten;
+					var remains = progress.stat.size - write.bytesWritten;
+					var percent = ((written / progress.stat.size) * 100).toPrecision(2);
+					opts.onProgress({
+						file: src,
+						written: written,
+						remains: remains,
+						percent: percent
```

Comment:
that would also allow to drop `remains`, since that would be very easy to calc it yourself.